src/main/kotlin/minesweeper/domain/MineMap.kt

```diff
@@ -0,0 +1,31 @@
+package minesweeper.domain
+
+class MineMap(private val width: Width, private val height: Height) {
+
+    val mineMap = List(width.getWidth()) {
+        TileColumn(List(height.getHeight()) { Tile(TileType.PLAIN) })
+    }
+
+    fun makeMine(mineCount: MineCount) {
+        require(width.getWidth() * height.getHeight() >= mineCount.getMineCount()) {
+            "지뢰는 넓이*너비의 수보다 작거나 같아야한다"
+        }
+        var makeCount = 0
+        do {
+            makeCount = setMineToTile(makeCount)
+        } while (makeCount != mineCount.getMineCount())
+    }
+
+    private fun setMineToTile(makeCount: Int): Int {
+        val rangeX = width.getWidth()
+        val rangeY = height.getHeight()
+        val position = RandomPosition.of(rangeX, rangeY)
+        val positionX = position.first
+        val positionY = position.second
+        val tile = mineMap[positionX][positionY]
```

Comment:
Pair 타입이라 아래처럼 사용할 수 있을 것 같네요.

```suggestion
        val (x, y) = position
        val tile = mineMap[x][y]
```

Reply:
말씀 주신 내용으로 수정해보았습니다.